api/renterhost_test.go

```diff
@@ -249,3 +249,111 @@ func TestIntegrationUploadDownload(t *testing.T) {
 		t.Fatal("the uploading is not succeeding for some reason:", rf.Files[0], rf.Files[1])
 	}
 }
+
+// TestIntegrationRenterMetrics tests that the renter's metrics are properly
+// reported after uploading and downloading.
+func TestIntegrationRenterMetrics(t *testing.T) {
```

Comment:
fine for now but we've dropped the 'Integration' and 'Unit' conventions, because we don't actually use them.